build/tasks/generate-imports.js

```diff
@@ -0,0 +1,83 @@
+/*eslint-env node */
+const UglifyJS = require('uglify-js');
+
+module.exports = grunt => {
+	grunt.registerMultiTask(
+		'generate-imports',
+		'Task for generating an axe.imports module with external dependencies.',
+		function() {
+			// Convenience method that utilises uglifyjs tree-transformer to unwrap umd module resolver
+			const removeUMD = new UglifyJS.TreeTransformer(node => {
```

Comment:
We should consider moving this to something like Babel (in the long-term). No need to change now.